Code/Framework/AzCore/AzCore/Jobs/Job.cpp

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <AzCore/base.h>
+#include <AzCore/Jobs/Job.h>
+#include <AzCore/Jobs/JobCancelGroup.h>
+#include <AzCore/Jobs/JobContext.h>
+#include <AzCore/Jobs/JobManager.h>
+#include <AzCore/std/parallel/atomic.h>
+
+AZ::Job::Job(bool isAutoDelete, AZ::JobContext* context, bool isCompletion, AZ::s8 priority)
```

Comment:
The AZ namespace should be openeded up and the functions placed into it without the AZ:: prefix.

Reply:
done